README.md

```diff
@@ -107,6 +107,42 @@ hatch run test
 hatch run test tests/test_<SOME_FILE>.py
 ```
 
+### Integration Tests
+
+```sh
+# Run all tests
+hatch run integration-test
+
+# Run a single test file
+hatch run integration-test integration-test/test_<SOME_FILE>.py
+```
+
+To run integration tests on your local machine, you need to set up your Kaggle API credentials. There are two ways to do this:
```

Comment:
Does reading the credentials from `kaggle.json` works for these tests locally? 

https://github.com/Kaggle/kagglehub?tab=readme-ov-file#option-3-read-credentials-from-kagglejson

Reply:
seems like it worked for me. adding it to the doc